frontend/src/lib/k8s/index.ts

```diff
@@ -39,53 +39,40 @@ import ServiceAccount from './serviceAccount';
 import StatefulSet from './statefulSet';
 import StorageClass from './storageClass';
 
-const classList = [
-  ClusterRole,
-  ClusterRoleBinding,
-  ConfigMap,
-  CustomResourceDefinition,
-  CronJob,
-  DaemonSet,
-  Deployment,
-  Endpoints,
-  LimitRange,
-  Lease,
-  ResourceQuota,
-  HPA,
-  PodDisruptionBudget,
-  PriorityClass,
-  Ingress,
-  IngressClass,
-  Job,
-  Namespace,
-  NetworkPolicy,
-  Node,
-  PersistentVolume,
-  PersistentVolumeClaim,
-  Pod,
-  ReplicaSet,
-  Role,
-  RoleBinding,
-  RuntimeClass,
-  Secret,
-  Service,
-  ServiceAccount,
-  StatefulSet,
-  StorageClass,
-];
-
-const resourceClassesDict: {
-  [className: string]: KubeObject;
-} = {};
-
-classList.forEach(cls => {
-  // Ideally this should just be the class name, but until we ensure the class name is consistent
-  // (in what comes to the capitalization), we use this lazy approach.
-  const className: string = cls.className.charAt(0).toUpperCase() + cls.className.slice(1);
-  resourceClassesDict[className] = cls;
-});
-
-export const ResourceClasses = resourceClassesDict;
+export const ResourceClasses = {
+  ClusterRole: ClusterRole,
+  ClusterRoleBinding: ClusterRoleBinding,
+  ConfigMap: ConfigMap,
+  CustomResourceDefinition: CustomResourceDefinition,
+  CronJob: CronJob,
+  DaemonSet: DaemonSet,
+  Deployment: Deployment,
+  Endpoint: Endpoints,
+  LimitRange: LimitRange,
+  Lease: Lease,
+  ResourceQuota: ResourceQuota,
+  HorizontalPodAutoscaler: HPA,
+  PodDisruptionBudget: PodDisruptionBudget,
+  PriorityClass: PriorityClass,
+  Ingress: Ingress,
+  IngressClass: IngressClass,
+  Job: Job,
+  Namespace: Namespace,
+  NetworkPolicy: NetworkPolicy,
+  Node: Node,
+  PersistentVolume: PersistentVolume,
+  PersistentVolumeClaim: PersistentVolumeClaim,
+  Pod: Pod,
+  ReplicaSet: ReplicaSet,
+  Role: Role,
+  RoleBinding: RoleBinding,
+  RuntimeClass: RuntimeClass,
+  Secret: Secret,
+  Service: Service,
+  ServiceAccount: ServiceAccount,
+  StatefulSet: StatefulSet,
+  StorageClass: StorageClass,
+};
```

Comment:
Why all these declared with both key and value?

Reply:
for better type safety. in other files when using something like `ResourceClasses.Pod` it will be correctly typed as a `Pod`. Otherwise it would just be `Record<string, KubeObject>` which erases all that info

Reply:
refactored where key and value are the same